compute/admin_guide/install/install_defender/install_serverless_defender.adoc

```diff
@@ -203,6 +203,38 @@ The steps for embedding Serverless Defender differ depending on the build tool.
 
 . Unzip the Serverless Defender bundle into your working directory.
 
+. Inside the `twistlock` directory (the root directory in the zip), create a new sub-directory with the following structure: `com/twistlock/serverless/defender/<version>/`. For example, for version 22.06.286:
+
+  mkdir -p com/twistlock/serverless/defender/22.06.286
+
+. Move the `twistlock*.jar` file into this new subdirectory.
+
+. Rename the *.jar file to the convention: defender-<version>.jar (e.g. defender-22.06.286.jar).
+
+. Create a file called defender-<version>-pom.xml in the same location of the jar (change the version tag based on your version):
+
+.. Define the internal (remote) repository in the `*pom.xml` file:
+
+  <project>
```

Comment:
Don't see this mentioned in the GH issue

Reply:
This code comes from this [DR](https://spring.paloaltonetworks.com/twistlock/twistlock/wiki/%5BDR%5D-%5BServerless-Defender%5D-Defender-injection-for-java-functions#internal-repository). Please suggest! 
Thanks

Reply:
The DR suggests a workaround for customers with older console versions (before Lagrange this feature was broken).
For customers using Lagrange - we fixed it in the BE, these manual steps are not required.